src/DependencyInjection/ShapecodeCronExtension.php

```diff
@@ -14,10 +15,16 @@ final class ShapecodeCronExtension extends Extension
     /**
      * @inheritDoc
      */
-    public function load(array $config, ContainerBuilder $container): void
+    public function load(array $configs, ContainerBuilder $container): void
     {
         $locator = new FileLocator(__DIR__ . '/../Resources/config');
         $loader  = new Loader\YamlFileLoader($container, $locator);
         $loader->load('services.yml');
+
+        $configuration = new Configuration();
+        $config        = $this->processConfiguration($configuration, $configs);
```

Comment:
This part could be removed if you change the extended class to `\Symfony\Component\HttpKernel\DependencyInjection\ConfigurableExtension`.